tools/autograd/derivatives.yaml

```diff
@@ -2083,6 +2084,19 @@
   result0: gather(self_t.flatten(-3), -1, result1.flatten(-3)).view_as(result1)
   output_differentiability: [True, False]
 
+#mps
+- name: mps_max_pool2d(Tensor self, int[2] kernel_size, int[2] stride=[], int[2] padding=0, int[2] dilation=1, bool ceil_mode=False) -> Tensor
```

Comment:
These can go when we update native_functions.yaml

Reply:
Added the comment and created an issue.